styleguide/template.md

```diff
@@ -129,20 +131,30 @@ When linking to an API, you can use its unique identifier (UID) that is auto-gen
 
 You can use one of the following syntax:
 
-1. Markdown link: `[link_text](xref:UID)`
-2. Auto link: `<xref:UID>`
-3. Shorthand form: `@UID`
+1. Auto-link: `<xref:UID>` or `<xref:UID?displayProperty=fullName>`
+
+ The displayProperty query parameter produces a fully qualified link text. Default behavior is to produce a link text that shows only the member or type name.
```

Comment:
a fully qualified --> fully qualified
Default behavior is to produce a link text that shows --> By default, link text shows

Reply:
I'd add that the UID equates to the fully qualified class and member name.

Also, should we add an example of how to make an auto-link for a generic type or method?

Reply:
We do show more complex links like generics below with the encoding tips. Would you like to add them here too?

Reply:
I think it's fine that they are below. I wrote this comment before reading that far down.